compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/test/roundtrip.mlir

```diff
@@ -272,3 +272,15 @@ func.func @workgroup() {
   }
   return
 }
+
+// -----
+
+// CHECK-LABEL: @reference_to()
+// CHECK:       %[[ALLOC:.+]] = memref.alloc()
+// CHECK:       amdaie.reference_to
+// CHECK-SAME:  %[[ALLOC]]
```

Comment:
Any reason to match `%[[ALLOC]]` use via `CHECK-SAME` ?

I believe it can be combined with the previous `CHECK` statements as `amdaie.reference_to %[[ALLOC]]` ?